pyscriptjs/src/main.ts

```diff
@@ -234,6 +262,37 @@ export class PyScriptApp {
         logger.info('All paths fetched');
     }
 
+    /**
+     * Fetches all the python plugins specified in this.config, saves them on the FS and import
+     * them as modules, executing any plugin define the module scope
+     *
+     * @param runtime - runtime that will execute the plugins
+     */
+    async fetchPythonPlugins(runtime: Runtime) {
+        const plugins = this.config.plugins;
+        logger.info("Python plugins to fetch: ", plugins)
+        for (const singleFile of plugins) {
+            logger.info(`  fetching plugins: ${singleFile}`);
+            try {
+                const pathArr = singleFile.split('/');
+                const filename = pathArr.pop();
+                // TODO: Would be probably be better to store plugins somewhere like /plugins/python/
+                //       or similar
+                const destPath = `./${filename}`
+                await runtime.loadFromFile(destPath, singleFile);
```

Comment:
Yes. Agree on both things... I'm on a rush right now so I won't change the sequential -> parallel aspected but it can be done as a follow up PR. In case no one takes care of it, I can do it when back after Thanksgiving